Bottom: 6be22fb12cd012007970e989a47c996ed8d0a55d
Top:    db2511e6438de79d7b2a6fb337fce701ea367a1b
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-07-18 10:49:52 -0700

Refresh of acm_msg

---

diff --git a/configure.ac b/configure.ac
index 59c5eb1..b4303c6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -70,6 +70,9 @@ AC_CHECK_HEADER(valgrind/memcheck.h, [],
 fi
 fi
 
+AC_CHECK_HEADER(infiniband/acm.h, 
+      AC_DEFINE([USE_IB_ACM], 1, [setting IBACM support]), [])
+
 dnl Checks close on exec support
 AC_CHECK_HEADERS([fcntl.h sys/socket.h])
